build.gradle

```diff
@@ -73,8 +73,8 @@ subprojects {
     mavenPublishing {
         coordinates("com.thousandeyes.sdk", project.name, project.version)
         pom {
-            name = 'ThousandEyes Java SDK - ${project.name}'
-            description = 'A set of Java client libraries for the [Thousandeyes v7 API](https://developer.cisco.com/docs/thousandeyes/v7/).'
+            name = "ThousandEyes Java SDK - ${project.name}"
```

Comment:
is it just the " vs `? 🤯

Reply:
yep! double-quoted strings can be Groovy Strings, which can contain evaluated expressions